### ISSUE TITLE
howto remap ctrl h to delete previous character?

### ISSUE THREAD

**Issue by digika-ding** [atuinsh/atuin#1077]
howto remap ctrl h to delete previous character?

**Comment by kohane27** [comment]
I'm trying to do the same and found your issue.

A quick search and I found https://github.com/ellie/atuin/issues/636 which led to https://github.com/ellie/atuin/blob/6e3304b485a0bab3d41ad5746cd607c8a8b3f38c/src/command/client/search/interactive.rs#L72

You can simply change the line and then compile the binary yourself.

Maybe we have have a flag to config this as well?

**Comment by kohane27** [comment]
As mentioned form the talk [atuin: magical shell history with Rust](https://youtu.be/gxjaHU7O3TY?t=246), `atuin` has a command-line interface, as documented in [atuin history list](https://atuin.sh/docs/commands/list). I am also used to the `fzf`'s [key-bindings-for-command-line](https://github.com/junegunn/fzf#key-bindings-for-command-line).

I found https://github.com/ellie/atuin/issues/68#issuecomment-1567410629 and it works perfect for me. Since we're using `fzf` interface, `C-h` to delete a character works.

**Comment by collinvandyck** [comment]
I'm also looking for a way for ctrl-h to delete the previous char instead of moving the cursor to the left (I'm not using fzf for reverse histo search).

**Comment by ompugao** [comment]
you may use my fork:
```sh
cargo install --force --git https://github.com/ompugao/atuin --branch feat/backspace-with-ctrl-h
```

diff:
https://github.com/atuinsh/atuin/compare/main...ompugao:atuin:feat/backspace-with-ctrl-h

**Comment by ellie** [comment]
closed in favour of #193 

it's on my list to address key bindings fairly soon 🙏
